docs/reference/aggregations/bucket/terms-aggregation.asciidoc

```diff
@@ -59,13 +59,15 @@ GET /_search
 {
     "aggs" : {
         "genres" : {
-            "terms" : { "field" : "genre" }
+            "terms" : { "field" : "genre" } <1>
         }
     }
 }
 --------------------------------------------------
 // CONSOLE
 // TEST[s/_search/_search\?filter_path=aggregations/]
+<1> `terms` aggregation needs a field of type `keyword` or any other data type suitable for bucket aggregations. In order to use it with `text` you will need to enable 
```

Comment:
Could we say `"should be a field of type 'keyword'"` instead of `"needs"` ?  Just because it doesn't _have_ to be a `keyword`, it's just strongly suggested/preferred :)

Reply:
Sure thing, fits better indeed. I'll update it as soon as I can.

Regarding the CLA, I used this email: kaisserds@gmail.com

I did get a confirmation with a copy of the signed document when I signed it, by the way.